RealEstate/apps/core/views.py

```diff
@@ -219,3 +222,138 @@ def _permission_check(self, request, role, *args, **kwargs):
 
     def get(self, request, *args, **kwargs):
         return render(request, self.template_name, {})
+
+
+class CategoryView(BaseView):
+    """
+    View for the Category Ranking Page.
+    """
+    template_name = 'core/categories.html'
+
+    def _permission_check(self, request, role, *args, **kwargs):
+        return True
```

Comment:
And now I am realizing this function does need to be protected.  Realtors should not be able to get there, or if they can, it should be read only.  Lets lock it down to homebuyers only for now.  The permission check method is for more specific permission checks, I have something for generic for Realtor/Homebuyer-only URLs.  Add the following line as a class-level attribute to CategoryView:
```python
_USER_TYPES_ALLOWED = User._HOMEBUYER_ONLY
```